bundle/src/test/resources/com/adobe/acs/commons/wcm/shared/impl/shared-properties-reference-provider/apps-content.json

```diff
@@ -0,0 +1,659 @@
+            "column": {
+              "jcr:primaryType": "nt:unstructured",
+              "sling:resourceType": "granite/ui/components/coral/foundation/container",
+              "items": {
+                "jcr:primaryType": "nt:unstructured",
+                "label": {
+                  "jcr:primaryType": "nt:unstructured",
+                  "name": "./label",
+                  "fieldLabel": "Button label",
+                  "sling:resourceType": "granite/ui/components/coral/foundation/form/textfield"
+                },
+                "link": {
+                  "jcr:primaryType": "nt:unstructured",
+                  "suffix": ".html",
+                  "rootPath": "/content",
+                  "name": "./link",
+                  "fieldLabel": "Link to",
+                  "sling:resourceType": "granite/ui/components/coral/foundation/form/pathfield"
+                }
+              }
+            }
+          }
+        }
+      },
+      "dialogglobal": {
+        "jcr:primaryType": "nt:unstructured",
+        "jcr:title": "Footer Global",
+        "sling:resourceType": "cq/gui/components/authoring/dialog",
+        "content": {
+          "jcr:primaryType": "nt:unstructured",
+          "margin": false,
+          "sling:resourceType": "granite/ui/components/coral/foundation/fixedcolumns",
+          "items": {
+            "jcr:primaryType": "nt:unstructured",
+            "column": {
+              "jcr:primaryType": "nt:unstructured",
+              "sling:resourceType": "granite/ui/components/coral/foundation/container",
+              "items": {
+                "jcr:primaryType": "nt:unstructured",
+                "label": {
+                  "jcr:primaryType": "nt:unstructured",
+                  "name": "./label",
+                  "fieldLabel": "Button label",
+                  "sling:resourceType": "granite/ui/components/coral/foundation/form/textfield"
+                },
+                "link": {
+                  "jcr:primaryType": "nt:unstructured",
+                  "suffix": ".html",
+                  "rootPath": "/content",
+                  "name": "./link",
+                  "fieldLabel": "Link to",
+                  "sling:resourceType": "granite/ui/components/coral/foundation/form/pathfield"
+                }
+              }
+            }
+          }
+        }
+      },
+      "footer.html": {
+        "jcr:primaryType": "nt:file",
+        "jcr:content": {
+          "jcr:primaryType": "nt:resource",
+          "jcr:lastModifiedBy": "admin",
+          "jcr:mimeType": "text/html",
+          ":jcr:data": 864
+        }
+      }
+    }
+  },
+  "content": {
+    "text": {
+      "jcr:primaryType": "cq:Component",
+      "jcr:createdBy": "admin",
+      "jcr:title": "Text (v2)",
+      "jcr:created": "Tue Oct 15 2019 14:00:38 GMT-0600",
+      "jcr:description": "Rich Text Section",
+      "cq:icon": "textLeft",
+      "componentGroup": ".core-wcm",
+      "cq:dialog": {
```

Comment:
I know the details of the `cq:dialog` are not hurting anything, but are they providing any necessary function for the test?  If not, my vote would be to omit those details since its a ton of irrelevant stuff to wade through and represents a cut in time of what this component looks like which will over time become obsolete.

Reply:
This is a general comment - would like to see the content of this file largely slimmed down to remove irrelevant nodes.